config/jobconfig.toml

```diff
@@ -147,6 +147,12 @@
 [MIME]
 "#Playtime Requirements" = 0
 "#Required Account Age" = 0
+"#Spawn Positions" = 1
+"#Total Positions" = 1
+
+[PARAMEDIC]
```

Comment:
Is there a reason we're doing this instead of just `[Paramedic]`, `["Atmospheric Technician"]`, etc?

Reply:
```dm
	/// String key to track any variables we want to tie to this job in config, so we can avoid using the job title. We CAPITALIZE it in order to ensure it's unique and resistant to trivial formatting changes.
	/// You'll probably break someone's config if you change this, so it's best to not to.
```

I wrote this in the DMDoc for `config_tag`, and I based it around the conversation I had with oranges when he was giving me a few examples on what he wanted the system to be resistant to, like if we changed it from `Station Engineer` to like `Station engineer` (this is the example he gave me), and I kinda wanted to emphasize that it was a key and not the actual title of the job by having that be `STATION_ENGINEER`. Let me know if that's bad UX or something.

Reply:
Fair enough